templates/content-widget-price-filter.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * The template for displaying product price filter widget.
+ *
+ * This template can be overridden by copying it to yourtheme/woocommerce/content-widget-price-filter.php
+ *
+ * HOWEVER, on occasion WooCommerce will need to update template files and you
+ * (the theme developer) will need to copy the new files to your theme to
+ * maintain compatibility. We try to do this as little as possible, but it does
+ * happen. When this occurs the version of the template file will be bumped and
+ * the readme will list any important changes.
+ *
+ * @see     https://docs.woocommerce.com/document/template-structure/
+ * @package WooCommerce/Templates
+ * @version 3.6.1
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+?>
+<?php do_action( 'woocommerce_widget_price_filter_item_start', $args ); ?>
```

Comment:
Maybe it is better to call this action `woocommerce_widget_price_filter_start` as this template doesn't represent an item?

Reply:
I followed the "pattern" I saw in other widget templates:

`woocommerce_widget_product_review_item_start`

`woocommerce_widget_product_item_start`

Reply:
IMHO, it would be better to remove `item` from the filter name because `content-widget-price-filter.php` is used to display the whole widget. In the case of the other two templates that you mentioned, they are used to display an item within the widget and not the whole widget. What do you think?

Reply:
I think it's fine. I'm changing it now.